doc/api/esm.md

```diff
@@ -241,10 +241,16 @@ There are four types of specifiers:
 Bare specifiers, and the bare specifier portion of deep import specifiers, are
 strings; but everything else in a specifier is a URL.
 
-Only `file://` URLs are supported. A specifier like
+Only `file:` and `data:` URLs are supported. A specifier like
 `'https://example.com/app.js'` may be supported by browsers but it is not
 supported in Node.js.
 
+`data:` URLs only resolve _Bare specifiers_ for builtin modules and
```

Comment:
Are the "bare specifier" and "absolute specifier" terms documented elsewhere? I don't know what they are.

Reply:
They are documented for [ES Modules' import mechanism as a whole](https://github.com/nodejs/node/blob/db6423179fa31c4b87a8f61d19abfe264edbdb75/doc/api/esm.md#terminology) which happened outside of this PR

Reply:
Looks good. Linking to the definition might be helpful.